src/log-plugin-version.js

```diff
@@ -1,10 +1,10 @@
-const { resolve } = require('path');
+const path = require('path');
 const readPkg = require('read-pkg');
 const debug = require('debug')('semantic-release:monorepo');
 
 const logPluginVersion = type => plugin => async (pluginConfig, config) => {
   if (config.options.debug) {
-    const { version } = await readPkg(resolve(__dirname, '../'));
+    const { version } = await readPkg(path.resolve(__dirname, '../'));
```

Comment:
What does this have to do with this PR?

Reply:
Not a lot, just thought I'd make the path imports consistent - but actually maybe I did it the wrong way round. I'll revert it.